packages/ses-ava/src/ses-ava-test.js

```diff
@@ -29,7 +27,7 @@ const isPromise = maybePromise =>
 const logErrorFirst = (func, args, name, logger = defaultLogger) => {
   let result;
   try {
-    result = apply(func, undefined, args);
+    result = func(...args);
```

Comment:
What’s the motivation for this simplification, or what was the motive for the complication? (I like the simplification)